packages/point-intercept/src/main.jsx

```diff
@@ -36,7 +36,7 @@ export class Main extends React.Component {
     };
   }
 
-  componentWillReceiveProps(nextProps) {
+  UNSAFE_componentWillReceiveProps(nextProps) {
```

Comment:
we'll leave this for now - but this change (and all the others not related to placement-ordering) should be in their own pr.